classes/admin/class-kp-form-fields.php

```diff
@@ -371,6 +371,67 @@ private static function kp_form_build_settings() {
 			)
 		);
 
+		/* Klarna Express Checkout (aka Express Button). */
+		$settings = array_merge(
+			$settings,
+			array(
+				'express_options'     => array(
+					'title' => 'Express Checkout (aka Express Button)',
+					'type'  => 'title',
+				),
+				'express_enabled'     => array(
+					'title'   => __( 'Show Express Button on cart page', 'klarna-payments-for-woocommerce' ),
+					'type'    => 'checkbox',
+					'label'   => __( 'Enable', 'klarna-payments-for-woocommerce' ),
+					'default' => 'no',
+
+				),
+				'express_data_theme'  => array(
+					'title'   => __( 'Theme', 'klarna-payments-for-woocommerce' ),
+					'type'    => 'select',
+					'default' => 'default',
+					'options' => array(
+						'default' => __( 'Default', 'klarna-payments-for-woocommerce' ),
+						'dark'    => __( 'Dark', 'klarna-payments-for-woocommerce' ),
+						'light'   => __( 'Light', 'klarna-payments-for-woocommerce' ),
+					),
+				),
+				'express_data_shape'  => array(
+					'title'   => __( 'Shape', 'klarna-payments-for-woocommerce' ),
+					'type'    => 'select',
+					'default' => 'default',
+					'options' => array(
+						'default' => __( 'Default', 'klarna-payments-for-woocommerce' ),
+						'rect'    => __( 'Rectangular', 'klarna-payments-for-woocommerce' ),
+						'pill'    => __( 'Pill', 'klarna-payments-for-woocommerce' ),
+					),
+				),
+				'express_data_label'  => array(
+					'title'   => __( 'Label', 'klarna-payments-for-woocommerce' ),
+					'type'    => 'select',
+					'default' => 'default',
+					'options' => array(
+						'default' => __( 'Default', 'klarna-payments-for-woocommerce' ),
+						'klarna'  => __( 'Klarna', 'klarna-payments-for-woocommerce' ),
+					),
+				),
+				'express_data_width'  => array(
```

Comment:
Change the desc_tip and placeholder to both be a description instead. For example
'description' => __( 'A value between 145 and 500 (measured in px). Leave blank for default width.', 'klarna-payments-for-woocommerce' ),